presto-main/src/main/java/com/facebook/presto/testing/TestingConnectorSession.java

```diff
@@ -65,10 +66,25 @@ public TestingConnectorSession(
             List<PropertyMetadata<?>> propertyMetadatas,
             Map<String, Object> propertyValues,
             boolean isLegacyTimestamp)
+    {
+        this(user, source, Optional.empty(), timeZoneKey, locale, startTime, propertyMetadatas, propertyValues, isLegacyTimestamp);
+    }
+
+    public TestingConnectorSession(
```

Comment:
How many callers does the above version have? I don’t like adding a bunch of variants. Better to just update the callers (or switch to a builder).

Reply:
Actually, there are not many. I will update the PR and the one depends on it.